src/middlewares/__tests__/user-authorization.test.js

```diff
@@ -0,0 +1,70 @@
+const userAuthorization = require('../user-authorization');
+
+const incorrectUserId = 'IncorrectUserId';
+const correctUserId = 'CorrectUserId';
+const mockUser = {
+  _id: 'CorrectUserId',
+  username: 'John',
+  email: '123@gmail.com',
+  password: '123456',
+};
+const res = {
+  status: jest.fn().mockReturnThis(),
+  json: jest.fn().mockReturnThis(),
+};
+
+afterAll(() => {
+  jest.resetAllMocks();
+});
+describe('user-authorization function middleware', () => {
+  it('Should authorize user when he tries to modify his information', () => {
```

Comment:
Let's keep our language gender-neutral. So you can say `Should authorize user when trying to modify own information`.